testsuite/MDAnalysisTests/test_tprparser.py

```diff
@@ -236,3 +238,64 @@ class TPR510(TPRBase):
 
 class TestTPR510(_TestTopology, TPR510):
     """Testing TPR version 103"""
+
+
+def _test_is_in_topology(name, elements, topology_section, topology_path):
```

Comment:
If you've gone to the effort of writing down all the bond names and passing them this far, you may as well include some sort of detailed error message, `"Bond type {} not found".format(name)`